Fix multiple spelling mistakes

ggoncalopereira opened the pull request:
Attempt to fix a bunch of spelling mistakes.
Couldn't test locally for any discrepancies (HTML ellipsis, apostrophes, quotes...) but I'll have to trust my gut on this one.

ggoncalopereira committed:
Fix multiple spelling mistakes

Attempt fix a bunch of spelling mistakes.
Couldn't test locally for any discrepancies (HTML ellipsis, apostrophes, quotes...) but I'll have to trust my gut on this one.

ggoncalopereira commented:
Also had to guess the meaning of some sentences, but hopefully the main essence of it is still present.

nelsonmestevao reviewed (changes requested):
We don't edit the `HTML` files directly. They are automatically generated from [hugo](https://gohugo.io/).

The source code for the website tutorial is found under [`src/`](https://github.com/cesium/fractal-tree/tree/master/src) and I believe the file you are looking for is [`_index.md`](https://github.com/cesium/fractal-tree/blob/master/src/content/_index.md).

ggoncalopereira committed:
Fix several spelling mistakes

Follow-up on the previous commit, but to this specific file

nelsonmestevao commented:
@ggoncalopereira can you drop the first commit? :pray:

ggoncalopereira committed:
Fix individual spelling mistake

Small nuance detected and thoroughly corrected

ggoncalopereira commented:
@nelsonmestevao my bad - updated the .md and only pushed the .HTML commit. Should be fixed soon

nelsonmestevao commented on src/content/_index.md:
```diff
@@ -6,20 +6,22 @@ weight: -270
-Now that you know what a fractal is, a fractal tree is simply a fractal that builds
-a tree as it is being created. Okay, after this basic explanation lets get it started!
+Now that you know what a fractal is, a fractal tree is simply a fractal that builds a
+tree as it is being created. Okay, after this basic explanation let&rsquo;s get it
```
Use `'` here, please :pray:

ggoncalopereira committed:
Update index.html

nelsonmestevao commented on src/content/_index.md:
```diff
@@ -94,9 +97,9 @@ In this example the canvas has a 200x200 dimension.
-Since a blank canvas is boring and programming is not, you need to give it a
-little bit of life. To do so, you use a background that allows editing your
-canvas color from 0 to 255, 0 being black and 256 white, respectively. Read
+Since a blank canvas is boring and programming is not, you need to give it a 
```
Remove the extra space (end of `a`) :pray:

nelsonmestevao commented on src/content/_index.md:
```diff
@@ -111,14 +114,13 @@ The background is now Black.
+With your "base" now set, let's learn how to draw. In this case we will be
+drawing lines - and what do you need to draw a line? Simple answer: exactly two
+points, no more, no less. So give the function two points (height, width) and let the `line` do its
+job by connecting those points. See? As easy as it can be. See more [`here`](https://p5js.org/reference/#/p5/line)
```
Please, try to keep the text under 80 characters :point_up:

ggoncalopereira committed:
Correct indentation

Fixes some tabs, ellipsis and quotes

ggoncalopereira commented:
LGTM, should hopefully be good to go @nelsonmestevao

nelsonmestevao commented on src/content/_index.md:
```diff
@@ -445,15 +444,15 @@ It still doesn't work.
-when the if condition makes you stop you pop, it means you go back to the state
-saved in push. Up to this point you have only done the right side of the tree
-now for the left side just push and pop again but this time to the negative
+ck to interfere with the left side. Instead it goes right and builds the
```
`ck` should be `back`

ggoncalopereira committed:
Correct indentation

Attempt to keep each sentence below 80 characters
Fix missing apostrophe and "chopped words" (thanks, Insert key)
Remove extra spaces

ggoncalopereira commented:
Latest commit should fix all those issues - also pushed back previous wrongly indented sentences to keep the margin of 80 characters, max.

ggoncalopereira commented:
Actually - one second, found a bunch more things...

ggoncalopereira committed:
Remove wrong indentation

Hyphen bullet-pointed a phrase when shortening the "characters per line"

ggoncalopereira commented:
Should be good now!

ggoncalopereira committed:
Update _index.md

Tiny spelling mistake solved

ggoncalopereira commented:
Should be good now!

ggoncalopereira commented:
@nelsonmestevao any updates on this? Pinging to ask, that way the repo can be deployed and/or used whenever

nelsonmestevao commented:
I am coming back to this asap :muscle:

ggoncalopereira committed:
Update _index.md

Fixed yet another paragraphing issue caused by indentation
